projectile.el

```diff
@@ -867,16 +867,31 @@ Files are returned as relative paths to the project root."
      ((eq vcs 'git) projectile-git-submodule-command)
      (t ""))))
 
-(defun projectile-get-all-sub-projects (project)
-  "Get all sub-projects for a given projects.
-PROJECT is base directory to start search recursively."
+(defun projectile-get-immediate-sub-projects (project)
+  "Get immediate sub-projects for a given project. Does not recurse.
+It is assumed that `project` will end with an appropriate path delimiter, such as '/' or a '\'"
   (let* ((default-directory project)
          ;; search for sub-projects under current project `project'
          (submodules (mapcar
                       (lambda (s)
-                        (file-name-as-directory (expand-file-name s default-directory)))
-                      (projectile-files-via-ext-command (projectile-get-sub-projects-command)))))
+                        (file-name-as-directory (expand-file-name s project)))
```

Comment:
Okay; sure, there's no reason to change from `default-directory` to `project`. No problem.